src/pages/Tournament/TournamentDetails/BasicInfoPanel/EditPanel.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { ChangeEvent, useState } from 'react';
```

Comment:
Codacy has a fix for the issue: [Replace `React,·{·ChangeEvent,·useState·}·from·'react` with `{·Button,·DatePicker,·Form·}·from·'antd`](https://app.codacy.com/gh/othello-organisation-singapore/othello-storm-system-frontend/pullRequest?prid=8062249)
```suggestion
import { Button, DatePicker, Form } from 'antd';
```